integration-tests/gradle/src/test/java/io/quarkus/gradle/HttpUtils.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.gradle;
+
+import io.quarkus.test.devmode.util.DevModeTestUtils;
+import io.quarkus.utilities.JavaBinFinder;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Future;
+import java.util.function.Supplier;
+
+public class HttpUtils {
+
+    private static Future<?> quarkusDev;
+
+    protected static Process doLaunch(Path jar, File output) throws IOException {
```

Comment:
This method doesn't conceptually belong here. Best move it to a `LaunchUtils` class and also rename the method to `launch`.

Reply:
Okayy